src/compute-client/src/controller.rs

```diff
@@ -1107,7 +1108,13 @@ impl<T: ComputeControllerTimestamp> InstanceState<T> {
     where
         F: FnOnce(&mut Instance<T>) + Send + 'static,
     {
-        self.client.send(Box::new(f));
+        let otel_ctx = OpenTelemetryContext::obtain();
+        self.client.send(Box::new(move |instance| {
+            let _span = debug_span!("instance.call").entered();
```

Comment:
Don't most other places use something like `instance::call`?

Reply:
Looks like it, I'll change it!